pyblazing/pyblazing/apiv2/context.py

```diff
@@ -2296,6 +2287,28 @@ def create_table(self, table_name, input, **kwargs):
                         kwargs["names"].pop(id)
                         kwargs["dtype"].pop(id)
 
+            # if the input is a directory and files do not have extension we want to raise an error
+            if name_file[-1] == "/" and extension == "":
+                all_files = os.listdir(name_file)
```

Comment:
Something is wrong here. If we are creating a table from a folder, we will get the files listed out by the C++ layer. I cant remember where or how that happens, but the python layer cant be the one doing it. If you have an HDFS or S3 or GCS filesystem, python cant do `os.listdir`